### ISSUE TITLE
Emily research - home page button issue

### ISSUE THREAD

**Issue by randyabramson** [18F/crt-portal#940]

**Comment by emilycryan** [comment]
We have two options for this - either linking both buttons to "/report/" instead of jumping down the page 
- or -
doing a slow-ish scroll down to the second button when the first/top button is clicked. 

This is a question for @timoballard and @Jkrzy from a dev standpoint - which is faster/easier - because both should make the experience better, regardless of how we handle it. 

One more thing - we DO need to change the label on the second button to match the first button, "Start a Report". Since we have the translations for that, it should be an easy change (aka, just make it match the top button) 

CC @blaughman @randyabramson @AvivaOskow

**Comment by timoballard** [comment]
Having them both link to `/report` would be very easy, and, as you say, wouldn't required any new translations.

The slow/smooth scrolling option is easy to implement using the [scroll-behavior CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior). Only downside is that some [browsers don't support it](https://caniuse.com/?search=scroll-behavior), notably IE and Safari. If we wanted the smooth scrolling to work everywhere, we'd have to implement a fallback scrolling effect in javascript

**Comment by emilycryan** [comment]
@timoballard Let's do the two buttons linking to /report/ then bc I'd prefer we have the same experience for any user vs different experiences based on browser. And updating the bottom button label to match the top for now. 

CC @blaughman @randyabramson @AvivaOskow

**Comment by randyabramson** [comment]
I agree...we should talk about other ways to get folks 'educated' on their rights (bigger font for NAV?), but I think the scrolling will frustrate those who just wanna start a report.

cc: @emilycryan @blaughman @timoballard

**Comment by blaughman** [comment]
Both buttons match in all languages in Dev.  Moving to UAT.

**Comment by randyabramson** [comment]
Tested on live site and the button went to the right URL on mobile and desktop.

Closing ticket.
